docs/scalar-dlt-compatibility-table.md

```diff
@@ -0,0 +1,17 @@
+# Scalar DLT Compatibility Table
+
+This document shows the compatibility of different scalar-ledger and scalardl-java-client-sdk versions.
+
+## Compatibility Table
+scalar-ledger | scalardl-java-client-sdk    |
```

Comment:
Why there are no `|` on the left?
It seems interpreted but I think it is invalid as a markdown?
https://www.markdownguide.org/extended-syntax/#tables

Reply:
Thank you for your review comment.
Table format updated accordingly.